package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-measures-data",
-  "version": "1.6.1",
+  "version": "1.6.2",
```

Comment:
should we not bump the version number? the readme says to always bump when measures data is changed, and otherwise optional; this change seems so invisible that we should skip a version bump (unless it's valuable to have a release for every change of any kind?)

Reply:
i should rtfm! good catch